cypress/integration/products/menageProducts/availableForPurchaseProducts.js

```diff
@@ -0,0 +1,124 @@
+import faker from "faker";
+
+import ProductSteps from "../../../steps/productSteps";
+import { urlList } from "../../../url/urlList";
+import ChannelsUtils from "../../../utils/channelsUtils";
+import FrontShopProductUtils from "../../../utils/frontShop/frontShopProductUtils";
+import ProductsUtils from "../../../utils/productsUtils";
+import ShippingUtils from "../../../utils/shippingUtils";
+
+// <reference types="cypress" />
+describe("Products available in listings", () => {
+  const shippingUtils = new ShippingUtils();
+  const channelsUtils = new ChannelsUtils();
+  const productsUtils = new ProductsUtils();
+  const productSteps = new ProductSteps();
+  const frontShopProductUtils = new FrontShopProductUtils();
+  const startsWith = "Cy-";
+  const name = `${startsWith}${faker.random.number()}`;
+  let productType;
+  let attribute;
+  let category;
+  let defaultChannel;
+  let warehouse;
+
+  before(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    shippingUtils.deleteShipping(startsWith);
+    productsUtils.deleteProperProducts(startsWith);
+
+    channelsUtils
+      .getDefaultChannel()
+      .then(channel => {
+        defaultChannel = channel;
+        cy.fixture("addresses");
+      })
+      .then(addressesFixture => {
+        shippingUtils.createShipping(
```

Comment:
Use object as a single param here as well, the additional benefit apart from what @mmarkusik mentioned is that it explains the arguments when invoking the function without having to look at the function definition